packages/calypso-build/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # [Unreleased]
 
+- Update `@automattic/wordpress-external-dependencies-plugin` references in `README.md` to `@wordpress/dependency-extraction-webpack-plugin`.
```

Comment:
This was part of 3.0.0, look a few lines lower 🙂

Reply:
Not quite :grimacing: I'm specifically calling out the changes to the README here -- which weren't in 3.0.0 :wink: 
(But it's debatable whether they're worth mentioning in the Changelog -- feel free to remove.)